docs/local_development.md

```diff
@@ -33,6 +33,13 @@ RSpec.describe YourClass, :determinator_support do
     it "should respond in a way that is defined by being out of the experiment"
   end
 
+  context "when the constraints are defined dynamically" do
+    forced_determination(:experiment_name, 'variant_b', only_for: constraints)
+    let(:constraints) { { employee: true } }
+
+    it "should respond in a way that shows employees the experiment"
```

Comment:
What do you think of changing this to:
```
it "should respond in a way that shows employees the experiment" do
# ...
end
```
Without the block i thought for longer than i would like to admit that `should respond in a way` is some weird RSpec syntax for shared examples 🙈